scipy/interpolate/tests/test_polyint.py

```diff
@@ -198,14 +198,14 @@ def test_high_derivative(self):
                                 np.zeros(len(self.test_xs)))
 
     def test_hermite(self):
-        xs = [0,0,0,1,1,1,2]
-        ys = [self.true_poly(0),
-              self.true_poly.deriv(1)(0),
-              self.true_poly.deriv(2)(0),
-              self.true_poly(1),
-              self.true_poly.deriv(1)(1),
-              self.true_poly.deriv(2)(1),
-              self.true_poly(2)]
+        # xs = [0,0,0,1,1,1,2]
+        # ys = [self.true_poly(0),
+        #       self.true_poly.deriv(1)(0),
+        #       self.true_poly.deriv(2)(0),
+        #       self.true_poly(1),
+        #       self.true_poly.deriv(1)(1),
+        #       self.true_poly.deriv(2)(1),
+        #       self.true_poly(2)]
         P = KroghInterpolator(self.xs,self.ys)
```

Comment:
@aarchiba What is the purpose of the `xs`, `ys` variables in this test? They appear to be unused  with `self.xs, self.ys` used instead.  Should the test use `xs`?  Or should  `xs`, `ys` be just completely removed? Thanks!

Reply:
Since they don't seem to be used previously, maybe simply remove it. Or add as a new separate test

Reply:
Removed.